Velocity_Model/basins.py

```diff
@@ -1,90 +1,31 @@
-    ]
-)
-v205_basin_outlines = v204_basin_outlines
-v206_basin_outlines = v205_basin_outlines + get_basin_full_path(
-    [
-        "Data/USER20_BASINS/SpringsJ_basin_outline_v1_WGS84.txt",
-        "Data/USER20_BASINS/Karamea_basin_outline_v1_WGS84.txt",
-        "Data/USER20_BASINS/CollingwoodBasinOutline_1_WGS84_v1.txt",
-        "Data/USER20_BASINS/CollingwoodBasinOutline_2_WGS84_v1.txt",
-        "Data/USER20_BASINS/CollingwoodBasinOutline_3_WGS84_v1.txt",
-    ]
-)
+with open(vm_data_dir/ "basins.yaml", "r") as yaml_file:
+    yaml_data = yaml.safe_load(yaml_file)
 
-# Wellington Basin update has the same outline so have not updated it for this
-v207_basin_outlines = v206_basin_outlines + get_basin_full_path(
-    [
-        "Data/Basins/Greater_Wellington_and_Porirua/v21p7/GreaterWellington1_Outline_WGS84.dat",
-        "Data/Basins/Greater_Wellington_and_Porirua/v21p7/GreaterWellington2_Outline_WGS84.dat",
-        "Data/Basins/Greater_Wellington_and_Porirua/v21p7/GreaterWellington3_Outline_WGS84.dat",
-        "Data/Basins/Greater_Wellington_and_Porirua/v21p7/GreaterWellington4_Outline_WGS84.dat",
-        "Data/Basins/Greater_Wellington_and_Porirua/v21p7/GreaterWellington5_Outline_WGS84.dat",
-        "Data/Basins/Greater_Wellington_and_Porirua/v21p7/GreaterWellington6_Outline_WGS84.dat",
-        "Data/Basins/Greater_Wellington_and_Porirua/v21p7/Porirua1_Outline_WGS84.dat",
-        "Data/Basins/Greater_Wellington_and_Porirua/v21p7/Porirua2_Outline_WGS84.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/HawkesBay1_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/HawkesBay2_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/HawkesBay3_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/HawkesBay4_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/Napier1_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/Napier2_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/Napier3_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/Napier4_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/Napier5_Outline_WGS84_delim.dat",
-        "Data/Basins/Napier_Hawkes_Bay/v21p7/Napier6_Outline_WGS84_delim.dat",
-        "Data/Basins/Gisborne/v21p11/Gisborne_Outline_WGS84_delim.dat",
-        "Data/Basins/Southern_Hawkes_Bay/v21p12/SHB_Outline_WGS84_delim.dat",
-        "Data/Basins/Wairarapa/v21p12/Wairarapa_Outline_WGS84_delim.dat",
-        "Data/Basins/East_Cape/v22p3/Motu_bay1_Outline_WGS84.txt",
-        "Data/Basins/East_Cape/v22p3/Motu_bay2_Outline_WGS84.txt",
-        "Data/Basins/East_Cape/v22p3/Motu_bay3_Outline_WGS84.txt",
-    ]
-)
-basin_outlines_dict = {
-    "2.03": v203_basin_outlines,
-    "2.04": v204_basin_outlines,
-    "2.05": v205_basin_outlines,
-    "2.06": v206_basin_outlines,
-    "2.07": v207_basin_outlines,
-}
+def get_basin_files_for_a_version(ver):
```

Comment:
Wouldn't this only get the current basins additional basin files? Not the previous basins as well?

Reply:
I misunderstood the purpose of this function, it's all good here.